scripts/build-dist.js

```diff
@@ -45,6 +45,8 @@ babelConfig.babelrc = false;
 babelConfig.presets = babelConfig.presets.map((preset) => {
     return preset === 'es2015' ? 'es2015-rollup' : preset;
 });
+babelConfig.plugins = babelConfig.plugins || [];
+babelConfig.plugins.push(['transform-react-remove-prop-types', {'mode': 'wrap'}]);
```

Comment:
The scripts are doing builds for both development and production. I'm suggesting that you do what's in the README for this Babel plugin: https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types#via-babelrc-recommended